impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Abstract.js

```diff
@@ -256,10 +271,94 @@ define([
 
       this._configureOptions();
 
+      this._chart.on("dblclick", function (params) {
+        this._onExecute(params);
+      }, this);
+
+      // When 'click' and 'dblclick' are bound to same element
+      // and user double-clicks, 'click' is fired twice followed by dbl-click
+      // Reference: https://stackoverflow.com/a/7119911
+      var clickTimeout;
+      this._chart.on("click", function (params) {
```

Comment:
Would this `params` variable be more accurately named as `ev` (for event)?

Reply:
The properties that `ev` usually holds is in `params.event` or `params.event.event`. For such cases, using `ev` felt weird. params also contain some properties ( like value, seriesName, dataIndex, componentType etc. ) that tell us more about the point that is clicked . 
For someone who is only reading code, won't `ev` appear as the usual event object? 😅

Reply:
Ok, so this is a modifed echarts event. Would `echartsEvent` be overkill? Not sure. Not bad, as is.